requirements.txt

```diff
@@ -3,3 +3,4 @@ ops == 2.5.1
 pydantic ==1.10.12
 psycopg2-binary ==2.9.7
 requests == 2.31.0
+cosl == 0.0.6
```

Comment:
Iirc, you just need this for testing. Charmcraft will pull it because it's present in the lib file

Reply:
I'm trying to remember what happened that I needed to add the cosl here...I'll try to reproduce.

Reply:
No idea. Removed, thanks.

Reply:
Found it after running a charmcraft clean

```
unit-teste1-0: 12:03:19 WARNING unit.teste1/0.install   File "/var/lib/juju/agents/unit-teste1-0/charm/./src/charm.py", line 22, in <module>
unit-teste1-0: 12:03:19 WARNING unit.teste1/0.install     from observability import Observability
unit-teste1-0: 12:03:19 WARNING unit.teste1/0.install   File "/var/lib/juju/agents/unit-teste1-0/charm/src/observability.py", line 8, in <module>
```
So I added again the cosl library to requirements

Reply:
Doesn't the trace originate in a test?

Reply:
No, teste-1 is the name of the application. It originates from "MetricsEndpointProvider" [here](https://github.com/canonical/synapse-operator/blob/main/src/observability.py#L8tp), from prometheus-scrape libpath 39. In a follow-up PR, I'll update the library again so will get the PYDEPS.